pootle/templates/user/profile.html

```diff
@@ -34,6 +34,10 @@ <h3 class="user-score">
       {% endif %}
     </h3>
 
+    <h3 class="user-detailed-stats">
+      <a href="{% url 'pootle-user-stats' object.username %}">{% trans 'User stats' %}</a>
```

Comment:
It might be worth to align the wording with the user menu and use _User statistics_ instead.